build.sbt

```diff
@@ -1,44 +1,57 @@
 import ReleaseTransformations._
 import microsites.ExtraMdFileConfig
 
+parallelExecution := false
```

Comment:
What's up with parallel execution? Also, does this line affect anything as it's not a part of any project?

Reply:
Oh, that's not needed, I'll take it out. I had in there cause one of the machines I develop on didn't quite have the horsepower to run all the tests at once and would lock up.